src/WCSimDetectorConfigs.cc

```diff
@@ -463,18 +463,18 @@ void WCSimDetectorConstruction::SetNuPrismBeamTest_mPMTGeometry()
 	vessel_cyl_height = 38.*CLHEP::mm;    //option A, option B would be 277 mm
 	vessel_radius_curv = 342.*CLHEP::mm;  //needs to include the vessel thickness, as we construct from outside inwards.
 	vessel_radius = 254.*CLHEP::mm;
-	dist_pmt_vessel = 2*CLHEP::mm;      
+	dist_pmt_vessel = 8*CLHEP::mm;      
```

Comment:
@nickwp Thanks for updating. Did you test compilation in this mode? 

Then should @laurenanthony2 @akutsuR try to run this branch to ensure it's working as expected?

Reply:
I tested compilation but not running and validating WCTE simulations